helm-deploy-test/helm-deploy-test.yml

```diff
@@ -0,0 +1,49 @@
+# Concourse pipeline to build deploy and teardown cf against a kube cluster
+
+resources:
+- name: cf-ci
+  type: git
+  source:
+    uri: https://github.com/SUSE/cf-ci.git
+    branch: master
+    paths:
+    - helm-deploy-test/*
+
+- name: s3.scf-config.linux
+  type: s3
+  source:
+    access_key_id: {{aws-access-key}}
+    secret_access_key: {{aws-secret-key}}
+    bucket: cf-opensusefs2
+    regexp: config/scf-linux-amd64-(.*).zip$
+
+- name: pool.k8s-hosts
+  type: pool
+  source:
+    uri: git@github.com:SUSE/cf-ci-pools.git
+    private_key: {{github-private-key}}
+    branch: k8s-hosts
+    pool: k8s-hosts
+
+jobs:
+- name: helm-deploy-test
+  plan:
+  - aggregate:
+    - get: cf-ci
+      trigger: true
+    - get: s3.scf-config.linux
+      trigger: true
+    - put: pool.k8s-hosts
+      params: {acquire: true}
+  - task: cf-deploy
+    file: cf-ci/helm-deploy-test/tasks/cf-deploy.yml
+  - task: cf-smoke-tests
+    file: cf-ci/helm-deploy-test/tasks/smoke-tests.yml
+  - task: acceptance-tests-brain
+    file: cf-ci/helm-deploy-test/tasks/acceptance-tests-brain.yml
+  - task: acceptance-tests
+    file: cf-ci/helm-deploy-test/tasks/acceptance-tests.yml
+  - task: cf-teardown
+    file: cf-ci/helm-deploy-test/tasks/cf-teardown.yml
+  - put: pool.k8s-hosts
```

Comment:
it needs comment in that case.
Reason explained: https://github.com/SUSE/cf-ci/pull/8
"so the idea is we will have test tasks after cf-deploy tasks. If they succeed only then the tear down task will run and in turn the release lock. Otherwise, the teardown and release lock wont happen, making sure that we look into the failure. Failure should not be a common thing and must inflict this pain of manual teardown and release lock in case failure happens. Thus, forcing us to an inquiry."